_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -371,7 +371,7 @@ The `table` is not [visible][] in page.
 The rule applies only to `headers` attribute within a `table` element.
 
 ```html
-<div role="table">
+<div role="table" aria-label="Projects status">
```

Comment:
```suggestion
<div role="table" aria-label="Grades">
```

Suggesting another name (better context, at least for me)
Also, while you're at it, can I suggest swapping the order of the cells so they become aligned with the headers. Just not to raise any questions about the order of presentation of the cells.